nekoyume/Assets/_Scripts/UI/Widget/Base/XTweenPopupWidget.cs

```diff
@@ -8,7 +8,13 @@ namespace Nekoyume.UI
     public abstract class XTweenPopupWidget : PopupWidget
     {
         [SerializeField]
-        private AnchoredPositionXTweener xTweener = null;
+        private AnchoredPositionSingleTweener xTweener = null;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            xTweener.axisType = AnchoredPositionSingleTweener.AxisType.X;
```

Comment:
This external component settings is better set in inspector than script. For non-programmers.

Reply:
`axisType` is already can control in inspector. The code is a device that was made to prevent mistakes.